### PR TITLE
Feature multi page review delete last page

### PR DESCRIPTION
**Pull request by a-szotyori**
Deleting the last page finishes the Multi Page Review Screen.

For "open with" imported images the user is prompted for confirmation.

### How to test
Run an example app.

### Ticket 
Issue #202

### CODE REVIEW

File: ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java [hunk 1]
```diff
@@ -359,8 +393,12 @@ private void deleteFromCaches(final ImageDocument document) {
     }
 
     private void updatePageIndicator(final int position) {
-        mPageIndicator.setText(String.format("%d von %d", position + 1,
-                mMultiPageDocument.getDocuments().size()));
+        final int nrOfDocuments = mMultiPageDocument.getDocuments().size();
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java]: `numberOfDocuments` or `documentsCount` are also ok! 😉

File: ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java [hunk 2]
```diff
@@ -359,8 +393,12 @@ private void deleteFromCaches(final ImageDocument document) {
...
+        String text = null;
+        if (nrOfDocuments > 0) {
+            text = String.format("%d von %d", position + 1, nrOfDocuments);
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java]: No localization for this string!

File: ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java [hunk 3]
```diff
@@ -372,6 +410,11 @@ private void updateReorderPagesTip() {
     }
 
     private void updateNextButtonVisibility() {
+        if (mMultiPageDocument.getDocuments().size() == 0) {
+            mButtonNext.setVisibility(View.INVISIBLE);
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java]: Invisible or disabled?

File: ginivision/src/main/java/net/gini/android/vision/review/multipage/thumbnails/ThumbnailsAdapter.java [hunk 4]
```diff
@@ -318,7 +319,7 @@ public static int getNewPositionAfterDeletion(final int deletedPosition, final i
         final int newPosition;
         if (deletedPosition == newSize) {
             // Last item was removed, highlight the new last item
-            newPosition = deletedPosition - 1;
+            newPosition = Math.max(0, deletedPosition - 1);
```
Comment (kikettas) [inline ginivision/src/main/java/net/gini/android/vision/review/multipage/thumbnails/ThumbnailsAdapter.java]: Am I the only one that read here _Mad Max_?

File: ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java (hunk 2 above)
Comment (a-szotyori) [reply ginivision/src/main/java/net/gini/android/vision/review/multipage/MultiPageReviewFragment.java]: 😱

**Pull request merged**